### ISSUE TITLE
java.lang.RuntimeException: AssigneeId 75459187248929 is not equal to user id

### ISSUE THREAD

**Issue by cunneen** [pablo127/asanaexporter#2]
```
$ java -jar ./build/libs/asanaexporter-all-1.0.0.jar '0/xxxxxxxxxxxxxxxxxxxxxxx'
 INFO [main] (Main.java:45) - Start importing tasks.
 INFO [main] (Main.java:50) - Imported 3581 tasks in 39 s.
 INFO [main] (Main.java:53) - Start generate csv.
ERROR [main] (Main.java:57) - Error occurred while running AsanaExporter.
java.lang.RuntimeException: AssigneeId 7511111111111111 is not equal to user id!
	at io.bitbucket.pablo127.asanaexporter.Main.getAssigneeName(Main.java:133)
	at io.bitbucket.pablo127.asanaexporter.Main.generateCsv(Main.java:87)
	at io.bitbucket.pablo127.asanaexporter.Main.main(Main.java:54)

```

(NB: I've changed the value of AssigneeId in the error message above)

**Comment by Fischmax** [comment]
@cunneen This is actually the same error I had in #1 .
Do you have multiple users in your Asana?
I'll try to complete issue #1 during the next days, that might solve your problem.

**Comment by cunneen** [comment]
Thanks, yes I do have multiple users. 

I looked at #1 briefly but not thoroughly enough; it does indeed seem to be the same issue. I had some success in logging a warning and returning "Unknown" instead of throwing a RuntimeException at [Main.java#133](https://github.com/pablo127/asanaexporter/blob/bd3524f9a188ac3d4d5b836231aba4ea8a562629/src/main/java/io/bitbucket/pablo127/asanaexporter/Main.java#L133).

However, I was looking for an exporter that included comments and attachments, and after successfully producing a CSV using the above means I realised that this package doesn't meet my needs.

Thanks again.
